### DISCUSSION TITLE
Bunkerweb Error

### DISCUSSION THREAD

**Discussion by kennysamuerto** [tteck/Proxmox#3525] [Help Needed]
Hi, I am trying to install Bunkerweb, and I have been able to install it several times without problems, but when I reinstall it now, I get an error:

 /bash: line 34: /etc/apt/keyrings/bunkerity_bunkerweb-archive-keyring.gpg: No such file or directory |(23) Failed writing body 

What can it be?

**Comment by sierralima123** [comment]
Error is raised by unmet dependencies for nginx.
During the installation process, nginx 1.26.2 will be installed. But bunkerweb depends on nginx 1.26.1.
I was not able to fix this issue by myself within the script (to adjust the depending version). Therefore, i've installed it on a new debian lxc with the installation procedure provided for bunkerweb linux integration.

**Reply by tteck** [reply]
Fixed https://github.com/tteck/Proxmox/commit/604b7ee6287c7e3afed143e85ef3951fac960048
